[wip] Add more C++ tests to Python wrapper

driazati opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#65581**
* #65514
* #65508

Differential Revision: [D31296507](https://our.internmc.facebook.com/intern/diff/D31296507)

driazati committed:
[wip] Add more C++ tests to Python wrapper

[ghstack-poisoned]

driazati committed:
Update on "[wip] Add more C++ tests to Python wrapper"




[ghstack-poisoned]

driazati committed:
Update on "[wip] Add more C++ tests to Python wrapper"




[ghstack-poisoned]

driazati commented:
@driazati has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31296507).

driazati committed:
Update on "[wip] Add more C++ tests to Python wrapper"


Differential Revision: [D31296507](https://our.internmc.facebook.com/intern/diff/D31296507)

[ghstack-poisoned]

peterjc123 commented:
@driazati test_api is called on Window at https://github.com/pytorch/pytorch/blob/master/.jenkins/pytorch/win-test-helpers/test_libtorch.bat#L15. Maybe you need to remove this one?

driazati committed:
Update on "[wip] Add more C++ tests to Python wrapper"


Differential Revision: [D31296507](https://our.internmc.facebook.com/intern/diff/D31296507)

[ghstack-poisoned]

driazati committed:
Update on "[wip] Add more C++ tests to Python wrapper"


Differential Revision: [D31296507](https://our.internmc.facebook.com/intern/diff/D31296507)

[ghstack-poisoned]

driazati committed:
Update on "[wip] Add more C++ tests to Python wrapper"


Differential Revision: [D31296507](https://our.internmc.facebook.com/intern/diff/D31296507)

[ghstack-poisoned]
